### ISSUE TITLE
TableRowSkeleton is not accessible for testing.

### ISSUE THREAD

**Issue by div-Leo** [freenowtech/wave#80]
**What**
The component `TableRowSkeleton` used as the loading state for the `Table` is not easy to access for testing purposes. 

**Why** 
The component renders a row with n columns without any text, aria-*, role, and querying the row (`td`) would not be reliable as it could be that is rendering the actual data instead.
